### PR TITLE
Bug 1128869 - Add support for running only one test file in grunt

### PR DESCRIPTION
**Pull request by crdlc**

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -59,7 +59,8 @@ module.exports = function(grunt) {
 
           // Generate the test file list automatically... Maybe there's an
           // easier way to do this.
-          urls: grunt.file.expand({}, [TEST_DIR + 'test_*.js']).map(
+          urls: grunt.file.expand({},
+                           [TEST_DIR + ('test_' + (grunt.option('fileName') || '*') + '.js')]).map(
```
Comment (AntonioMA) [inline Gruntfile.js]: Nit: I would prefer calling this parameter testFile (instead of fileName).

File: README.md [hunk 2]
```diff
@@ -61,6 +61,12 @@ then you need to specify the JSON reporter. The easiest way to do this is with:
 
 grunt test --testReporter=JSON
 
+How to run only one test file?
+
+grunt test --fileName=<name> (test_<name>.js)
```
Comment (AntonioMA) [inline README.md]: And if you change the parameter, change this also :)

BTW, the way this is written the first time I read this I thought that (test_<name>,js) was part of the command line.

Maybe rewrite this as:

If you want to run only the test contained on test_name.js, execute:

grunt test --testFile=name

e.g. to run only the test in test_join_room.js:

grunt test --testFile=join_room

Comment (crdlc) [inline README.md]: much better, thanks

File: Gruntfile.js (hunk 1 above)
Comment (crdlc) [inline Gruntfile.js]: done

**Pull request merged**